protocol/src/main/java/org/dousi/DousiServer.java

```diff
@@ -29,14 +35,23 @@ public DousiServer(ServerConfig serverConfig) {
    * Register the service into this exporter.
    *
    * @param interfaceClass The interface that we want to export.
-   * @param serviceObject The object that this service implementation.
+   * @param serviceObject  The object that this service implementation.
```

Comment:
Why change this line?

Reply:
idea format. I have reverted this change.